### ISSUE TITLE
Redesign site?

### ISSUE THREAD

**Issue by AlecDusheck** [ephs/ephs.github.io#4]
As mentioned in [this](https://github.com/ephs/ephs.github.io/issues/3#issuecomment-422240178) issue, I proposed recreating the site (since we are ditching the Github website and not using their platform anymore). 

Anyone have any platform ideas? I was thinking about nginx + [Angular 6](https://angular.io/) for frontend. This would allow for cool page transitions and easy expansion.

Maybe this would be a good project for when we start having regular meetings?

**Comment by andrewgazelka** [comment]
I agree. This is a great project for new people to work on. I am definitely for NGINX—I use it on most of my dedis. I am not that familiar with Angular 6, but I don't really use JS that much anyway, so I'm down for using it. For the backend want to use Django? I've used it on a few projects, and it has seemed to treat me quite nice. It is undoubtably a lot better than PHP, and I'm pretty sure more people here know Python than Ruby if we were going the Rails route.
